TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -859,6 +859,7 @@ def makeTreeFromMiniAOD(self,process):
         suff='',
         storeProperties=2,
         SkipTag=SkipTag,
+        storePileupIds=True
```

Comment:
change to this line is no longer needed since a member variable is now being used